### PR TITLE
Add a found bug list

### PR DESCRIPTION
**Pull request by wcshim40**
To show the bugs and CVEs found by _UTopia_, it seems good to have this page for multiple purposes.
- It shows the effect of the fuzzing automation approach.
- It provides insight into making unit tests that can work well with _UTopia_ (i.e., Fuzzing friendly unit tests).
- It increases the visibility of various fuzz drivers so that people can learn or improve fuzz driver composing strategies.

### CODE REVIEW

**Review by sole2 — COMMENTED** [review]
Thank you for adding bug list. It looks good idea :+1: 
One minor request before merge.
Would you add link in README to find it easier?

File: Trophy.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# 🏆 Bugs & CVEs 
+
+The showcase of bugs and CVEs found by *UTopia*-based fuzzers serves a variety of purpose:
+- Show the effect of the fuzzing automation approach (i.e.,*UTopia* can help find more bugs in the projects that are already registered and have been being fuzz tested in OSS-Fuzz)
+- Provide insight into making unit tests that can work well with *UTopia* (i.e., Fuzzing friendly unit tests)
+- Increase the visibility of various fuzz drivers so that people can learn or improve fuzz driver composing strategies
+
+Note that The merged fuzz drivers are different from the original ones generated by *UTofia* because they are reviewed and modified by the community.
```
Comment (yuweol) [inline Trophy.md]: typo: The merged -> the merged
typo: UTofia -> UTopia

File: Trophy.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Project | Fuzzer | Bugs or CVEs | Category 
+----- | ----------- | ------ | ---------
+[OpenCV](https://github.com/opencv/opencv) | [readnetfromtensorflow_fuzzer.cc](https://github.com/google/oss-fuzz/pull/7781) | [#21947](https://github.com/opencv/opencv/issues/21947) | `bof`
+
+
+
+
+### Description of categories:
+
+* `arith`: Arithmetic error (e.g., overflows, divide-by-zero)
```
Comment (yuweol) [inline Trophy.md:19]: Checks. We may not cover all categories, since UTopia generated fuzzers will find bugs based on address sanitizer.

**Review by yuweol — COMMENTED** [review]
Thanks. This format looks good. Please check my comments.

**Comment by wcshim40** [comment]
> Thank you for adding bug list. It looks good idea 👍 One minor request before merge. Would you add link in README to find it easier?

Done! Thanks for your important comment.

File: Trophy.md (hunk 1 above)
Comment (wcshim40) [reply Trophy.md]: 👍

File: Trophy.md (hunk 2 above)
Comment (wcshim40) [reply Trophy.md:19]: You're right, it's not a complete or necessary category. Let's start with this incomplete category and polish them further. I removed

**Review by sole2 — APPROVED** [review]
LGTM :+1:

**Review by musangk — APPROVED** [review]
LGTM 👍

**Pull request merged**